docs/Usage/0_installation.rst

```diff
@@ -1,52 +1,26 @@
+.. _Installation:
+
 ************
 Installation
 ************
 
-We use Apache Maven build automation tool.
-
-Using command line
-==================
-
-Install Maven on the computer.
-``sudo apt install maven``
-
-Clone the Dialog Manager repository.
-``git clone https://github.com/Roboy/roboy_dialog``
-
-Navigate to the root module.
-``cd roboy_dialog``
-
-Download roboy parser.
-``git submodule update --init ./roboy_parser``
-
-Install roboy parser as described in the roboy_parser docs or the 'semantic parser' page in 'INTERFACES AND SCOPE'.
-
-Compile the project - Maven will take care of the rest.
-``mvn compile``
-
-Execute the project.
-``mvn exec:java -Dexec.mainClass="roboy.dialog.ConversationManager"``
+In order to install the dialog system, one must clone it from GitHub::
 
+    git clone https://github.com/Roboy/roboy_dialog -b master --recursive
 
-Using IDE (Eclipse, Intellij IDEA)
-==================================
+A working version of Maven is required to build and resolve dependencies. Please refer to the ``README.md`` of `master <https://github.com/Roboy/roboy_dialog/blob/master/README.md>`_ or `devel <https://github.com/Roboy/roboy_dialog/blob/devel/README.md#installation>`_ for further installation instructions.
 
-Clone the Dialog Manager repository.
-``git clone https://github.com/Roboy/roboy_dialog``
+Changing the default configuration, for example the input/output is explained in :ref:`Configuration`.
 
-Now, import Dialog System as a Maven project into the IDE of your choice. Build and execute using ``roboy.dialog.ConversationManager`` as the main class.
 
-Using a telegram bot for I/O
-============================
+.. _Optional steps:
 
-Register a bot as described on the `telegram website`_.
-.._telegram website: https://core.telegram.org/bots#3-how-do-i-create-a-bot
+Optional steps
```

Comment:
imo the installation of the telegram API should be moved to the tutorials and it is not an optionall installation step @SezerCan @polinominom @Mireu

Reply:
Moved it to User Manual -> Configuration in b4971d9c369e22b2d2f0d77b649c34b6542891f6 .